samples/objc/FirebaseUI-demo-objc/Samples/Auth/FUIAuthViewController.m

```diff
@@ -279,21 +310,9 @@ - (NSString *)getAllIdTokens {
 
 - (void)signOut {
   NSError *error;
-  FIRUser *currentUser = self.authUI.auth.currentUser;
-  // If the user is anonymous, delete the user to avoid dangling anonymous users.
-  if (currentUser.isAnonymous) {
-    [currentUser deleteWithCompletion:^(NSError * _Nullable error) {
```

Comment:
Do we no longer need to delete anonymous users?

Reply:
We do, I just moved the code to the tableview cell. The reason is that I wanted to use the sign in button to trigger a sign-in when the current user was anonymous. Essentially I am separating anonymous sign in/out with "normal" sign in/out.